tools/generate-models/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env zsh
+
+MODE_VERIFY="verify"
+MODE_GENERATE="generate"
+MODE="$1" # verify or generate
+TARGET_FILE="$(pwd)/$2"
+
+# get git reference
+if [[ $MODE == $MODE_VERIFY ]]; then
+    TARGET_LINE=$(tail -n 1 $TARGET_FILE)
+    SHA_REGEX='[0-9a-f]{5,40}'
+    if [[ $TARGET_LINE =~ $SHA_REGEX ]]; 
+    then
+        GIT_REF=$MATCH
+    else
+        echo "Target SHA could not be read in $TARGET_FILE !"
+        return 1;
+    fi
+elif [[ $MODE == $MODE_GENERATE ]]; then
+    GIT_REF="master"
+else
+    echo "Invalid command!"
+    echo "Model generation: run.sh generate ${TARGET_LOCATION_FOR_NEW_FILE}"
+    echo "Model verification: run.sh verify ${FILE_TO_BE_VERIFIED}"
+    return 1;
+fi
+
+echo "Git ref:"
+echo $GIT_REF
+
+# fetch rum-events-format for given git ref
+SCRIPT_LOC=$(dirname "$0")
+cd $SCRIPT_LOC
+if [[ ! -d "rum-events-format" ]]; then
+    git clone git@github.com:DataDog/rum-events-format.git
+fi
+cd "rum-events-format"
+git fetch origin $GIT_REF
+git checkout FETCH_HEAD
+SHA=$(git rev-parse HEAD)
+cd ..
+
+# install quicktype and generate model
+if [[ ! -a $(npm bin)/quicktype ]]; then
+    npm install buranmert/quicktype
```

Comment:
yes but first i need to talk to opensource people about this because that fork is not for public use
i think `npm` has private packages, maybe i can publish this as a private `npm` package

Reply:
EDIT: private packages don't seem to be what we need. i will move my fork to DD org once opensource people approves ⏳

Reply:
done ✅ 
https://github.com/DataDog/quicktype